lib/shift4/configuration.rb

```diff
@@ -3,22 +3,53 @@
 module Shift4
   class Configuration
     class << self
-      attr_accessor :secret_key
+      attr_accessor :secret_key, :merchant
       attr_reader :api_url, :uploads_url
+
+      @api_url = 'https://api.shift4.com'
+      @uploads_url = 'https://uploads.api.shift4.com'
+
+      def api_url=(api_url)
+        @api_url = if api_url.nil?
+                     'https://api.shift4.com'
+                   else
+                     api_url.gsub(%r{/$}, "")
+                   end
```

Comment:
to wyglada jak by bylo 4 razy uzyte - idzie tu jakis utility method podstawic z ternary operatorem?